compiler/ballerina-lang/src/main/java/io/ballerina/projects/DocumentContext.java

```diff
@@ -58,7 +58,7 @@ class DocumentContext {
     private NodeCloner nodeCloner;
     private final DocumentId documentId;
     private final String name;
-    private final String content;
```

Comment:
IIRC, the design here is to make everything immutable, Even though this fixes the OOM issue, not sure the design here is correct. @sameerajayasoma @azinneera